### PR TITLE
initial health quest swagger

### PR DESCRIPTION
**Pull request by massrb**
Initial swagger for health quest service

## Original issue(s)
https://github.com/department-of-veterans-affairs/va.gov-team/issues/14625

### CODE REVIEW

File: modules/health_quest/app/docs/health_quest/v0/health_quest.yaml [hunk 1]
```diff
@@ -0,0 +1,251 @@
+openapi: '3.0.0'
+info:
+  version: 0.0.1
+  title: Health Quest
+  description: |
+    ## The API for the Healthcare Clipboard 
+
+    These endpoints include the Healthcare Experience Questionaire
```
Comment (kfrz) [inline modules/health_quest/app/docs/health_quest/v0/health_quest.yaml:8]:
```suggestion
    These endpoints include the Healthcare Experience Questionnaire
```

File: modules/health_quest/app/docs/health_quest/v0/health_quest.yaml [hunk 2]
```diff
@@ -0,0 +1,251 @@
+openapi: '3.0.0'
+info:
+  version: 0.0.1
+  title: Health Quest
+  description: |
+    ## The API for the Healthcare Clipboard 
+
+    These endpoints include the Healthcare Experience Questionaire 
+    Responses and clipboard related endpoints into the MAP PGD and scehduled appointments
```
Comment (kfrz) [inline modules/health_quest/app/docs/health_quest/v0/health_quest.yaml:9]:
```suggestion
    Responses and clipboard related endpoints into the MAP PGD and scheduled appointments
```

What is `MAP PGD` ? Please define initialisms on first invocation

File: modules/health_quest/app/docs/health_quest/v0/health_quest.yaml [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+    description: VA.gov API production environment
+    variables:
+      version:
+        default: v0
+paths:
+  '/appointments':
+    get:
+      tags:
+        - appointments
+      summary: Returns alist of online scheduling appointments
```
Comment (kfrz) [inline modules/health_quest/app/docs/health_quest/v0/health_quest.yaml:34]:
```suggestion
      summary: Returns a list of online scheduling appointments
```

File: modules/health_quest/app/docs/health_quest/v0/health_quest.yaml [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+      operationId: getAppointments
+      security:
+        - bearerAuth: []
+      responses:
+        '200':
+          description: Systems retrieved successfully
```
Comment (kfrz) [inline modules/health_quest/app/docs/health_quest/v0/health_quest.yaml:40]: What systems were retrieved, and why are we retrieving systems, and not `Appointments`, with the `getAppointments` operation?

File: modules/health_quest/spec/request/apidocs_request_spec.rb [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+RSpec.describe 'vaos appointment requests', type: :request do
```
Comment (kfrz) [inline modules/health_quest/spec/request/apidocs_request_spec.rb:4]: Is this a `VAOS` appointment request? I don't understand the naming discrepancy between "health_quest" and this spec

**Review by kfrz — CHANGES_REQUESTED** [review]

**Comment by kfrz** [comment]
@massrb Please link this PR with an existing issue for background, thanks!

**Comment by dillo** [comment]
@kfrz Hi, I'm the new backend developer for the Healthcare team. I will be taking over this PR and addressing all of the highlighted concerns.

File: modules/health_quest/app/docs/health_quest/v0/health_quest.yaml (hunk 4 above)
Comment (dillo) [reply modules/health_quest/app/docs/health_quest/v0/health_quest.yaml]: This was wrongfully set to Systems. I've set it to Appointments to correct the error.

File: modules/health_quest/spec/request/apidocs_request_spec.rb (hunk 5 above)
Comment (dillo) [reply modules/health_quest/spec/request/apidocs_request_spec.rb]: I've removed vaos and set it to health quest. This should remove any confusions and or discrepancies.

**Comment by dillo** [comment]
@kfrz I've addressed all PR comments and rebased master. This is ready for final review. Thanks.

**Review by LindseySaari — APPROVED** [review]

**Pull request merged**